src/ExtensionsSample/Samples/EasyTableSamples.cs

```diff
@@ -0,0 +1,83 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using ExtensionsSample.Models;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.EasyTables;
+using Microsoft.WindowsAzure.MobileServices;
+
+namespace ExtensionsSample
+{
+    public static class EasyTableSamples
+    {
+        // POCO output binding 
+        //   The binding inserts the newly created item into the Easy Table when the 
+        //   method successfully exits.         
+        public static void InsertItem([EasyTable] out Item newItem)
```

Comment:
Recommend removing this method and the host.Call - I don't think it adds much as a sample.

Fine with you making EasyTables conditional in Program.cs - just make sure I don't have to create a MS to run the samples :)